arelle/WebCache.py

```diff
@@ -113,6 +113,7 @@ def __init__(
                 self.encodeFileChars = re.compile(r'[<>:"\\|?*^]')
             else:
                 self.encodeFileChars = re.compile(r'[:^]')
+        self.builtInCacheDir = os.path.abspath(os.path.join(__file__, '..', 'resources', 'cache'))
```

Comment:
I'm assuming this doesn't work in the frozen builds.

[distro.py needs to be configured](https://github.com/Arelle/Arelle/blob/d210baf36f5c161852639c2169d8c5fb0cb1ba03/distro.py#L16-L24) to include the files and then the correct path has to be [determined based on the environment](https://github.com/Arelle/Arelle/blob/d210baf36f5c161852639c2169d8c5fb0cb1ba03/arelle/Cntlr.py#L169-L170).